.circleci/config.yml

```diff
@@ -170,10 +154,8 @@ jobs:
       - teardown_vm
   'Validator Tests':
     executor:
-      name: amphtml-xlarge-executor
+      name: validator-xlarge-executor
```

Comment:
Yet another thing I realized from looking at [this job](https://app.circleci.com/pipelines/github/ampproject/amphtml/12884/workflows/1453de09-26f5-44f9-bcfe-31b593e71fb5/jobs/206959): Today, the C++ validator tests are triggered only when validator code is changed. Since this PR isn't exercising that path, it might be worth triggering all tests once to make sure there are no surprises.

One way to do so is to push a temporary commit that touches any of our `package.json` files (doing so will trigger _all_ CI checks). Alternatively, merge this PR and observe the `main` branch push build, which will run all tests.

Reply:
I tested running all targets and it passes: https://app.circleci.com/pipelines/github/ampproject/amphtml/13010/workflows/8b70cd3e-d6e3-466a-a422-4d9752a7c038